src/js/view/utils/views-manager.js

```diff
@@ -56,17 +56,19 @@ function scheduleResponsiveRedraw() {
 
 function onOrientationChange() {
     views.forEach(view => {
-        if (view.model.get('visibility') >= 0.75) {
-            const state = view.model.get('state');
+        const { model } = view;
```

Comment:
I wouldn't recommend destructuring when only one argument is extracted; it incurs a small transpiler cost